model.ipynb

```diff
@@ -32,14 +32,15 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "The goal is to infer the volume fraction of a colloidal structural color sample from its reflectance spectrum. The generative model must be able to reproduce an experimental reflectance spectrum from a set of parameters defining the sample. The foundation of the generative model is a stochastic multiple scattering algorithm included in a separate python package at https://github.com/manoharan-lab/structural-color. Here, we describe the multiple scattering model, our generative model (and its uncertainties), and our likelihood function."
+    "The goal of this package is to infer the volume fraction of a colloidal structural color sample from its reflectance spectrum. The generative model must be able to reproduce an experimental reflectance spectrum from a set of parameters defining the sample. The generative model is a stochastic multiple scattering algorithm included in a separate python package at https://github.com/manoharan-lab/structural-color. Here, we describe the multiple scattering model, our generative model (and its uncertainties), and our likelihood function."
```

Comment:
I think this is not correct. The generative model makes use of structcol. Here it sounds like the generative model is entirely composed of structcol calculations.